cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLQueryUtils.java

```diff
@@ -52,52 +52,44 @@
      * @param timestampFormat the timestamp format
      * @param sqlInstance     the sql instance
      * @param destination     the destination
-     * @param connection      the connection
-     * @param attrNativeTypes the attr native types
-     * @return the prepared statement
-     * @throws SQLException the sql exception
+     * @return the string buffer
      */
-    public static PreparedStatement upsertStatement (LinkedHashMap<String, ArrayList<JsonElement>> aggregation,
-                                            LinkedHashMap<String, ArrayList<JsonElement>> lastData,
-                                            String tableName,
-                                            String tableSuffix,
-                                            String uniqueKey,
-                                            String timestampKey,
-                                            String timestampFormat,
-                                            String sqlInstance,
-                                            String destination,
-                                            Connection connection,
-                                            boolean attrNativeTypes) throws SQLException {
-
-
-        String query = sqlUpsertQuery(aggregation,
-                lastData,
-                tableName,
-                tableSuffix,
-                uniqueKey,
-                timestampKey,
-                timestampFormat,
-                sqlInstance,
-                destination).toString();
+    protected static StringBuffer sqlUpsertQuery(LinkedHashMap<String, ArrayList<JsonElement>> aggregation,
+                                                        LinkedHashMap<String, ArrayList<JsonElement>> lastData,
```

Comment:
not sure if indentation is fine.

Reply:
Fixed in [e0aa884](https://github.com/telefonicaid/fiware-cygnus/pull/1944/commits/e0aa8847b5596ace1a4bf45340f279f7aa8e90e0)